content/general/concepts/software-development-life-cycle/terms/iterative/iterative.md

```diff
@@ -1,16 +1,13 @@
  ---
-Title: 'Iterative Approach to Software Development'
+Title: 'Iterative Approach'
```

Comment:
```suggestion
Title: 'Iterative Model'
```

Reply:
We just decided to update all of our SDLC entries to use a naming convention that includes "model" in the title and filenames. Please update the folder and filename to: iterative-model.